app/views/refine/inline/criteria/index_advanced.html.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  # a hash mapping Category => [array, of, conditions], sorted by category
```

Comment:
I'm toying with in the future moving this whole thing into a method in the controller. Its a direct copy of the same block in index.html.erb